static/js/leaderboard.js

```diff
@@ -64,14 +94,24 @@ var LeaderboardRow = {
             <td class="l-col">{{(run.play_time).toFixed(3)}} s</td>
             <td>{{run.path.length}}</td>
 
-            <td style="min-width:400px">
+            <td class="col-lg">
                 {{run.path | pathArrow}}
                 <a v-if="!lobbyId" v-bind:href="'/replay?run_id=' + run.run_id" target="_blank" title="Replay" >
                     <i class="bi bi-play"></i>
                 </a>
             </td>
 
-
+            <td class="col">
+                <div 
+                    v-if="run.finished && 
+                    run.run_id === currentRunId && 
```

Comment:
`$this.parent` is a bit yucky

Reply:
Yeah I originally passed it in as a prop but wasn't sure if it made sense for a `LeaderboardRow` component